runtime/cere-dev/src/lib.rs

```diff
@@ -126,7 +126,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 30800,
+	spec_version: 30900,
```

Comment:
Dev updates should increment two lower digits only.

Reply:
Discussed.